net/mosh/Portfile

```diff
@@ -38,12 +37,7 @@ depends_lib             port:ncurses \
                         port:p${perl5.major}-getopt-long \
                         port:p${perl5.major}-io-socket-ip
 
-patchfiles              network-bind.patch \
-                        patch-version-subdir.diff
-
-# Remove patch-version-subdir.diff and autoreconf when mosh > 1.3.2 is released.
 use_autoreconf          yes
```

Comment:
`use_autoreconf yes` should no longer be necessary (and if it is, you need to not get rid of the args either)